daffodil-propgen/src/main/scala/org/apache/daffodil/propGen/TunableGenerator.scala

```diff
@@ -193,7 +200,7 @@ class PrimitiveTunable(name: String, schemaType: String, schemaDefault: String)
   }
 
   private val scalaDefault = schemaType match {
-    case "xs:string" => "\"" + schemaDefault + "\""
+    case "xs:string" if (name != "tempFilePath") => "\"" + schemaDefault + "\""
     case _ => schemaDefault
```

Comment:
It's a little confusing to have the tempFilePath logic split in two places. What about just having here
```scala
case "xs:string" if (name == "tempFilePath") => "System.getProperty(\"java.io.tempdir\")"
case "xs:string" => "\"" + schemaDefault + "\""
```

Reply:
You're right, will fix